core/common/src/main/java/alluxio/metrics/aggregator/SingleTagValueAggregator.java

```diff
@@ -68,7 +68,7 @@ public SingleTagValueAggregator(MetricsSystem.InstanceType instanceType, String
       Map<String, String> tags = metric.getTags();
       if (tags.containsKey(mTagName)) {
         String ufsName = MetricsSystem.getClusterMetricName(
-            mMetricName + "." + tags.get(mTagName));
+            mMetricName + "." + mTagName + Metric.TAG_SEPARATOR + tags.get(mTagName));
```

Comment:
Should we be using `Metric.getMetricNameWithTags`?

Reply:
done